<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QDialog" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>756</width>
    <height>1025</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>756</width>
    <height>1025</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>756</width>
    <height>1025</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>mainwindow_title</string>
  </property>
  <widget class="QWidget" name="mainLayout">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>756</width>
     <height>1025</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2" rowstretch="0">
    <property name="leftMargin">
     <number>8</number>
    </property>
    <property name="topMargin">
     <number>8</number>
    </property>
    <property name="rightMargin">
     <number>8</number>
    </property>
    <property name="bottomMargin">
     <number>8</number>
    </property>
    <property name="horizontalSpacing">
     <number>6</number>
    </property>
    <property name="verticalSpacing">
     <number>0</number>
    </property>
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="general">
       <attribute name="title">
        <string>General</string>
       </attribute>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>731</width>
          <height>521</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="audio_source_label">
           <property name="styleSheet">
            <string notr="true"/>
           </property>
           <property name="text">
            <string>Audio Source</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="audio_source_dropdown"/>
         </item>
         <item>
          <widget class="QLabel" name="source_language_label">
           <property name="text">
            <string>Source Language</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="source_language_dropdown">
           <item>
            <property name="text">
             <string>English</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>한국어</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>翻訳のみ</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>両方</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="translate_language_label">
           <property name="text">
            <string>Translate Language</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="translate_language_dropdown">
           <item>
            <property name="text">
             <string>English</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>한국어</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>翻訳のみ</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>両方</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="captions_view_label">
           <property name="text">
            <string>Captions View</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="captions_view_dropdown">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <item>
            <property name="text">
             <string>None</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Source Only</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Translation Only</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Both</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="profanity_filter_label">
           <property name="text">
            <string>Profanity Filter</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="profanity_filter_dropdown">
           <item>
            <property name="text">
             <string>Off</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>On</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="output_to_label">
           <property name="text">
            <string>Output to</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="output_to_dropdown">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <item>
            <property name="text">
             <string>Streams only</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="caption_timeout_label">
           <property name="text">
            <string>Caption Timeout (Seconds)</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="caption_timeout_spin">
           <property name="minimum">
            <number>15</number>
           </property>
           <property name="maximum">
            <number>60</number>
           </property>
           <property name="singleStep">
            <number>5</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="horizontalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>900</y>
          <width>729</width>
          <height>36</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QToolButton" name="toolButton">
           <property name="text">
            <string>...</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="cancel_button">
           <property name="styleSheet">
            <string notr="true">color: rgb(252, 252, 252);
border-radius: 8px;
border-style: outset;
background-color: rgb(217, 217, 217);
padding: 5px;</string>
           </property>
           <property name="text">
            <string>Cancel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="save_button">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">color: rgb(252, 252, 252);
border-radius: 8px;
border-style: outset;
background-color: rgb(40, 40, 40);
padding: 5px;</string>
           </property>
           <property name="text">
            <string>Save</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="format_text">
       <attribute name="title">
        <string>Format Text</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="text_filtering">
       <attribute name="title">
        <string>Text Filtering</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="authentication">
       <attribute name="title">
        <string>Authentication</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
